### PR TITLE
German translation and maintainer

### PR DESCRIPTION
**Pull request by S1SYPHOS**
Hey there,
maybe it's too soon, but I'm constantly working on this. Let me know what you think!

### CODE REVIEW

**Comment by jlevy** [comment]
:+1: Thanks -- yes, would be glad to have a German translation! Do you think you can maintain it as well?

**Comment by S1SYPHOS** [comment]
I do think so, yes!

**Comment by gernd** [comment]
Just asking: Is this still under development? If so, @S1SYPHOS would you need help?

**Comment by jlevy** [comment]
Would love to have a full German translation and maintainer.
@S1SYPHOS if you think you can resume, perhaps you can collaborate with @gernd to get a quality translation complete?
If not, @gernd I'd be open to a new PR (the English doc has changed quite a bit at this point in any case).

**Comment by S1SYPHOS** [comment]
I'm under alot of stress right now, there's little time, but I want to resume asap. Would love to work with @gernd !

**Comment by gernd** [comment]
@S1SYPHOS  ok cool. then my proposal would is that
* you choose a chapter that you have not translated yet
* assign me an issue  for it
* I work on it and give you a pull request for your branch

sounds reasonable?

**Comment by S1SYPHOS** [comment]
sure thing, first thing tomorrow evening ;)

**Comment by gernd** [comment]
@S1SYPHOS perfect! I will be on vacation until January 2nd. After that I will start.

So long and "Guten Rutsch" :tada:

**Comment by S1SYPHOS** [comment]
Well, in that case, I'll be on vacation until January 4th! New Year - fresh start, looking forward to it!

**Comment by gernd** [comment]
@S1SYPHOS which chapter should I translate?

**Comment by S1SYPHOS** [comment]
start from the bottom, I'll start at the top :)

**Comment by jlevy** [comment]
@S1SYPHOS @gernd Any updates on this? Would love to see this catch up with the current English version, verify it's ready to merge, and officially assign a maintainer, if one of you is willing.

**Comment by S1SYPHOS** [comment]
I'm on it

**Comment by S1SYPHOS** [comment]
I'm willing to maintain it, and hereby confirm that it's ready to merge.

File: README-de.md [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+
+- Um kürzlich genutzte Befehle zu sehen, `history`. Es gibt außerdem viele Abkürzungen wie etwa `!$` (letzter Parameter) und `!!` (letzter Befehl), wenngleich diese oft einfach ersetzt werden durch **ctrl-r** und **alt-.**.
+
+- Um lange Befehle zu bearbeiten, kannst du sie (nachdem du deinen Editor angegeben hast, etwa mit `export EDITOR=vim`) mit **ctrl-x** **ctrl-e** im Editor öffnen, um mehrere Zeilen bearbeiten zu können. Oder vi-Style, **escape-v**.
+
+- Um kürzlich verwendete Befehle anzuzeigen, benutze `history`. Anschließend `!n` (wobei `n` die Nummer des Befehls ist), um es erneut auszuführen. Es gibt zudem zahlreiche Abkürzungen, die man verwenden kann, die nützlichste ist wahrscheinlich `!$` für den letzten Parameter und `!!` für den letzten Befehl (siehe "HISTORY EXPANSION" in der `man`-Seite). Diese werden allerdings oft einfach ersetzt durch **ctrl-r** und **alt-.**.
+
+- Go to your home directory with `cd`. Access files relative to your home directory with the `~` prefix (e.g. `~/.bashrc`). In `sh` scripts refer to the home directory as `$HOME`.
```
Comment (jlevy) [inline README-de.md]: Still English?

File: README-de.md [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+
+- Eine andere Möglichkeit, ein wenig Unix auf dein Windows-System zu bringen, bietet [Cash](https://github.com/dthree/cash). Beachte allerdings, dass nur sehr wenige Unix-Befehle und Kommandozeilen-Optionen in dieser Umgebung zur Verfügung stehen.
+
+- Eine weitere Option, GNU-Entwicklerwerkzeuge (etwa GCC) auf Windows zu nutzen, besteht darin, [MinGW](http://www.mingw.org/) und dessen Paket [MSYS](http://www.mingw.org/wiki/msys), das Hilfsprogramme wie bash, gawk, make and grep beinhaltet, zu installieren. MSYS bringt allerdings nicht so viele Features mit wie etwa Cygwin. MinGW ist besonders nützlich für die Erstellung nativer Windows-Ports von Unix-Werkzeugen.
+
+
+## Weitere Quellen
+
+- [awesome-shell](https://github.com/alebcay/awesome-shell): Eine hilfreiche Liste von Shell-Werkzeugen und Quellen.
+- [awesome-osx-command-line](https://github.com/herrbischoff/awesome-osx-command-line): A more in-depth guide for the OS X command line.
```
Comment (jlevy) [inline README-de.md]: More English

**Comment by jlevy** [comment]
@S1SYPHOS great! I notice a bit of English still; perhaps overlooked? Will let you look again before I merge.

**Comment by S1SYPHOS** [comment]
Well, thanks for checking, it'll be done by tomorrow.

**Comment by S1SYPHOS** [comment]
@jlevy Alright, should be fine now, double-checked it 👍

**Comment by gernd** [comment]
thanks @S1SYPHOS for the correction!

**Comment by S1SYPHOS** [comment]
Thanks for the opportunity to contribute to this awesome project ;)

**Comment by S1SYPHOS** [comment]
@jlevy I took the liberty of adding @gernd and me to the authors-info.yml, I hope you don't mind!

**Comment by S1SYPHOS** [comment]
Great!

**Comment by jlevy** [comment]
All done. @S1SYPHOS you're maintainer now and will ask you to review German PRs.
@gernd of course your help is much appreciated and review helps as well! Thanks to you both!

**Comment by S1SYPHOS** [comment]
Have a good day, or, given you're in Berkeley, good evening 🌖

**Pull request merged**